sources/core/objects/MedbayTools.cpp

```diff
@@ -0,0 +1,59 @@
+#include "MedbayTools.h"
+
+HealthAnalyzer::HealthAnalyzer(quint32 id) : Item(id)
+{
+    SetSprite("icons/device.dmi");
+    SetState("health");
+}
+
+void HealthAnalyzer::Scan(IdPtr<Human> target)
+{
+    int health_digit = target->GetHealth() % 10000;
+    int health = (target->GetHealth() - health_digit) / 100;
+    int suffocation_digit = target->GetSuffocationDamage() % 10000;
+    int suffocation_damage = (target->GetSuffocationDamage() - suffocation_digit) / 100;
+    int burn_digit = target->GetBurnDamage() % 10000;
+    int burn = (target->GetBurnDamage() - burn_digit) / 100;
+    int brute_digit = target->GetBruteDamage() % 10000;
+    int brute = (target->GetBruteDamage() - brute_digit) / 100;
```

Comment:
Because it is only representation of data, `float` or `double` may be used. It will be much simpler here for percent.